test/e2e/README.md

```diff
@@ -208,60 +208,66 @@ Migration examples:
-      STANDBY_CLUSTER_CLOUD_PROVIDER=aws \ 
-      STANDBY_CLUSTER_OBJECT_STORE_PROVIDER=aws \
-      GINKGO_FOCUS=Migration
     ```
 
 ## Filtering tests
 
-Velero E2E tests uses [Ginkgo](https://onsi.github.io/ginkgo/) testing framework which allows a subset of the tests to be run using the [`-focus` and `-skip`](https://onsi.github.io/ginkgo/#focused-specs) flags to ginkgo.
+In release-1.15, Velero bumps the [Ginkgo](https://onsi.github.io/ginkgo/) version to [v2](https://onsi.github.io/ginkgo/MIGRATING_TO_V2).
+Velero E2E start to use [labels](https://onsi.github.io/ginkgo/#spec-labels) to filter cases instead of [`-focus` and `-skip`](https://onsi.github.io/ginkgo/#focused-specs) parameters.
 
-For filtering tests, using `make` variables `GINKGO_FOCUS` and `GINKGO_SKIP`  :
-1. `GINKGO_FOCUS`: Dot-separated list of labels to be included for Ginkgo description-based filtering. Optional. The `-focus` flag is passed to ginkgo using the `GINKGO_FOCUS` `make` variable. This can be used to focus on specific tests. 
-1. `GINKGO_SKIP`: Dot-separated list of labels to be excluded for Ginkgo description-based filtering.Optional. The `-skip ` flag is passed to ginkgo using the `GINKGO_SKIP` `make` variable. This can be used to skip specific tests.
+Both `make run-e2e` and `make run-perf` CLI support using parameter `GINKGO_LABELS` to filter test cases.
 
+`GINKGO_LABELS` is interpreted into `ginkgo run` CLI's parameter [`--label-filter`](https://onsi.github.io/ginkgo/#spec-labels).
 
 
-`GINKGO_FOCUS`/`GINKGO_SKIP` can be interpreted into multiple `-focus`/`-skip ` describe in [Description-Based Filtering](https://onsi.github.io/ginkgo/#description-based-filtering:~:text=Description%2DBased%20Filtering) by dot-separated format for test execution management please refer to examples below.:
-
-
-For example, E2E tests can be run with specific cases to be included and/or excluded using the commands below:
+### Examples
+E2E tests can be run with specific cases to be included and/or excluded using the commands below:
 1. Run Velero tests with specific cases to be included:
     ```bash
+    GINKGO_LABELS="Basic && Restic" \
+    CLOUD_PROVIDER=aws \
+    BSL_BUCKET=example-bucket \
+    CREDS_FILE=/path/to/aws-creds \
     make test-e2e \
-      GINKGO_FOCUS =Basic\][\Restic  \
-      CLOUD_PROVIDER=aws BSL_BUCKET=<BUCKET_FOR_E2E_TEST_BACKUP> BSL_PREFIX=<PREFIX_UNDER_BUCKET> CREDS_FILE=/path/to/aws-creds
     ```
-    In this example, only case  `[Basic][Restic]` is included.
+    In this example, only case have both `Basic` and `Restic` labels are included.
 
 1. Run Velero tests with specific cases to be excluded:
     ```bash
-    make test-e2e \
-      GINKGO_SKIP=Scale.Schedule.TTL.Upgrade\]\[Restic.Migration\][\Restic  \
-      CLOUD_PROVIDER=aws BSL_BUCKET=<BUCKET_FOR_E2E_TEST_BACKUP> BSL_PREFIX=<PREFIX_UNDER_BUCKET> CREDS_FILE=/path/to/aws-creds
+    GINKGO_LABELS="Scale || Schedule || TTL || (Upgrade && Restic) || (Migration && Restic)" \
```

Comment:
GINKGO_LABEL is interpret into ginkgo run so I don't think the change is right here.. because previously line 259 says GINKGO_SKIP.

Reply:
`GINKGO_LABELS` is interpreted as the `--label-filter` parameter for `ginkgo run`.

The `GINKGO_SKIP` and `GINKGO_FOCUS` parameters are interpreted to `ginkgo` CLI's `--focus` and `--skip` parameters.

Reply:
Right. So if you want to skip something by label, don't you have to have exclaimation mark `!<label>` like that?

> 
ginkgo --label-filter="!slow" | Avoid any specs labelled slow
-- | --

[ref](https://onsi.github.io/ginkgo/#spec-labels:~:text=ginkgo%20%2D%2Dlabel%2Dfilter%3D%22!slow,any%20specs%20labelled%20slow)

Reply:
@kaovilai that sounds right. Note "!restic" at line 73.

Reply:
Thanks. Modified.